CONTRIBUTING.md

```diff
@@ -44,3 +44,16 @@ To clean up build artefacts, cached analyzations, etc., run:
 ```
 npm run clean
 ```
+
+# Benchmarking
+
+A benchmark script for measuring the performance is available. Use it by running
+
+```
+node ./scripts/benchmark.mjs [tag = experimental] [reps = 1]
+
```

Comment:
measuring performance of what? and how? :)

Reply:
It is measuring the time from running `npx git-truck` and `npm i -g git-truck && git-truck` to Git Truck printing help information in the terminal.

Reply:
I've changed the build process to bundle all the dependencies of git truck which improves installation time significantly. I have new numbers incoming in a few minutes.

Reply:
Feel free to run the scripts now and post your results! Would be interesting to compare across systems.